### ISSUE TITLE
Openmm6.3 CUDA 7.0 test fails

### ISSUE THREAD

**Issue by ppxasjsm** [pandegroup/openmm#1151]
One of the test fails:
```
./TestCudaNonbondedForce double
exception: Error loading CUDA module: CUDA_ERROR_NO_BINARY_FOR_GPU (209)
```
should I be worried about this?
Since I have gtx980ti cards, I need the 352 driver:
```+------------------------------------------------------+                       
| NVIDIA-SMI 352.39     Driver Version: 352.39         |                       
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 980 Ti  Off  | 0000:02:00.0     Off |                  N/A |
| 22%   36C    P0    67W / 250W |     20MiB /  6142MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   1  GeForce GTX 980 Ti  Off  | 0000:03:00.0     Off |                  N/A |
| 22%   34C    P0    66W / 250W |     20MiB /  6143MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   2  GeForce GTX 980 Ti  Off  | 0000:81:00.0     Off |                  N/A |
| 22%   35C    P0    66W / 250W |     20MiB /  6143MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   3  GeForce GTX 980 Ti  Off  | 0000:82:00.0     Off |                  N/A |
|  0%   34C    P0    52W / 250W |     20MiB /  6143MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID  Type  Process name                               Usage      |
|=============================================================================|
|  No running processes found                                                 |
```

**Comment by peastman** [comment]
This was fixed a while ago by #1051.  It should work correctly in the latest version of the code.

No, you don't need to worry about it.  This was an error that only happened in double precision mode, when one dimension of the PME grid was exactly 49.  So pretty rare in practice.  And it lead to throwing an exception, so no risk of producing incorrect results or anything like that.

**Comment by ppxasjsm** [comment]
thanks for clarifying!
